packages/@aws-cdk/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -129,10 +129,11 @@ export abstract class FlowLogDestination {
   /**
    * Use S3 as the destination
    */
-  public static toS3(bucket?: s3.IBucket): FlowLogDestination {
+  public static toS3(bucket?: s3.IBucket, subfolder?: string): FlowLogDestination {
```

Comment:
rename subfolder -> keyPrefix 

Before we go ahead and add this property to the API, I want to make sure this is no under modeled. It might make sense to add `keyPrefix` to the `IBucket` interface. @iliapolo  what do you think?

Reply:
I think this is ok for now, but at as some point we should create an `S3Location` class and refactor all across the codebase.

Reply:
I'm not sure how many resources actually accept `bucketArn` + `prefix` outside of flow logs. I think it is a relatively unique scenario